### PR TITLE
Bump version to 2.1.2 & Added support for PHP 8.2

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Bump version to 2.1.2 & Added support for PHP 8.2
| Type?         | improvement
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | N/A
| How to test?  | No Need QA

### CODE REVIEW

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -27,6 +27,9 @@ jobs:
       - name: PHP syntax checker 8.1
         uses: prestashop/github-action-php-lint/8.1@master
 
+      - name: PHP syntax checker 8.2
+        uses: prestashop/github-action-php-lint/8.1@master
```
Comment (nicosomb) [inline .github/workflows/php.yml]: `8.2@master`

File: .github/workflows/php.yml [hunk 2]
```diff
@@ -58,7 +61,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        presta-versions: ['1.7.1.2', '1.7.2.5', '1.7.3.4', '1.7.4.4', '1.7.5.1', '1.7.6', '1.7.7', '1.7.8', 'latest']
+        presta-versions: ['1.7.1.2', '1.7.2.5', '1.7.3.4', '1.7.4.4', '1.7.5.1', '1.7.6', '1.7.7', '1.7.8', '8.0', 'latest']
```
Comment (nicosomb) [inline .github/workflows/php.yml:64]: And what about `8.1` ?

**Review by nicosomb — CHANGES_REQUESTED** [review]

File: .github/workflows/php.yml (hunk 1 above)
Comment (Progi1984) [reply .github/workflows/php.yml]: Fixed

File: .github/workflows/php.yml (hunk 2 above)
Comment (Progi1984) [reply .github/workflows/php.yml:64]: The `8.1` doesn't exist... It is `latest`, if I remember.

Comment (nicosomb) [reply .github/workflows/php.yml:64]: haha I thought it was about PHP.

**Review by nicosomb — APPROVED** [review]

**Comment by jolelievre** [comment]
Thanks @Progi1984

**Pull request merged**